### DISCUSSION TITLE
Is there any part of this project that embodies cloud-edge collaboration

### DISCUSSION THREAD

**Discussion by miku8miku** [kubeedge/ianvs#347] [Q&A] [answered]
I'm a beginner. Is there any part of this project that embodies cloud-edge collaboration?The existing code seems to run natively.

**Comment by MooreZheng** [comment, marked as answer]
At present, ianvs is used to develop and test cloud-edge collaborative algorithm in a local server. The issue has been discussed in https://github.com/kubeedge/sedna/issues/275. A simulator with multiple nodes is available last year with ianvs (https://github.com/kubeedge/ianvs/pull/35, https://github.com/kubeedge/ianvs/pull/39).
